pkg/apis/crds/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -312,3 +312,295 @@ spec:
     storage: true
```

Comment:
Shouldn't storage be false for v1alpha1?

Do we have tests that actually apply both of these CRDs? I _think_ the CRD won't be apply-able if storage is set to true for both.

Reply:
Maybe only the integration tests will hit that issue as the other tests use envtest?

Reply:
You def caught a bug!